### ISSUE TITLE
Kubernetes version always 1.66 althrouth having set 'orchestratorRelease'

### ISSUE THREAD

**Issue by zackliu** [Azure/acs-engine#1483]
**Is this a request for help?**:
YES
---

**Is this an ISSUE or FEATURE REQUEST?** (choose one):
ISSUE
---

**What version of acs-engine?**:
0.5.0
---

**Orchestrator and version (e.g. Kubernetes, DC/OS, Swarm)**
Kubernetes

**What happened**:
I define my json definition according to `examples/kubernetes-releases/kubernetes1.7.json`.
However, the created k8s cluster is still v1.66
Related definition:
```json
{
  "apiVersion": "vlabs",
  "properties": {
    "orchestratorProfile": {
      "orchestratorType": "Kubernetes",
      "orchestratorRelease": "1.7"
    },
    "masterProfile": {
      "count": 1,
      "vmSize": "Standard_D2_v2"
    },
    "agentPoolProfiles": [
      {
        "name": "agentpool",
        "count": 2,
        "vmSize": "Standard_D2_v2",
        "availabilityProfile": "AvailabilitySet"
      }
    ],
    "linuxProfile": {
      "adminUsername": "xxx",
      "ssh": {
        "publicKeys": [
          {
            "keyData": "xxx"
          }
        ]
      }
    },
    "servicePrincipalProfile": {
      "servicePrincipalClientID": "xxx",
      "servicePrincipalClientSecret": "xxx"
    }
  }
}

```

**What you expected to happen**:
Created k8s cluster in the latest v1.7.x or at least v1.7

**How to reproduce it** (as minimally and precisely as possible):
Use my definition to deploy a cluster

**Anything else we need to know**:

**Comment by jackfrancis** [comment]
Hi @zackliu we only rolled out support for the `orchestratorRelease` in `v0.6.0`. (We've also released `v0.7.0` since then.)

Do you need to be on `v0.5.0` of acs-engine?

**Comment by zackliu** [comment]
Resolved, thanks.
